usr/src/uts/common/fs/zfs/zfs_vfsops.c

```diff
@@ -2276,6 +2277,28 @@ zfs_get_zplprop(objset_t *os, zfs_prop_t prop, uint64_t *value)
 	return (error);
 }
 
+/*
+ * Return true if the coresponding vfs's unmounted flag is set.
+ * Otherwise return false.
```

Comment:
maybe explain what this is used for?  I think you're making it so that you can unmount the filesystem while we are in the middle of a free_long_range, by making the free_long_range bail out when we try to unmount?  So this is really telling us that we've started unmounting or are trying to unmount?